app/javascript/react/containers/AmusementParksIndexContainer.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import ParkTile from '../components/ParkTile'
+
+class AmusementParksIndexContainer extends React.Component {
+  constructor(props){
+    super(props)
+    this.state = {
+      amusementParks: []
+    }
+  }
+
+  componentDidMount(){
+    fetch('/api/v1/amusement_parks')
+    .then(response => {
+      if (response.ok) {
+        return response
+      } else {
+        let errrorMessage = `${response.status} (${response.statusText})`,
+          error = new Error(errrorMessage)
+        throw(error)
+      }
+    })
+    .then(response => response.json())
+    .then(parks => {
```

Comment:
the `parks` variable here is a bit awkward. I would suggest a `resp` variable for now.

The reason: `parks.amusement_parks` reads kind of weird.